flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSink.java

```diff
@@ -1245,6 +1246,12 @@ else if (scheme != null && authority == null) {
 			}
 
 			fs.initialize(fsUri, finalConf);
+
+			// By default we don't perform checksums on Hadoop's local filesystem and use the raw filesystem.
```

Comment:
The "by default" is not necessary anymore. We now always use the raw filesystem. This is a leftover from the previous version that allowed changing this.